modules/functionalities.js

```diff
@@ -0,0 +1,83 @@
+// grab essentail elements from document
+const todoItems = document.querySelector('.todo-items');
+const form = document.querySelector('.input-form');
+const newItem = document.querySelector('.form-data');
+
+const items = JSON.parse(localStorage.getItem('TodoList')) || [];
+
+// function to load items dynamically
+const loadItems = () => {
+  todoItems.innerHTML = ' ';
+  items.forEach((item) => {
+    const content = `
+        <li class="flex-item" id="${item.index}">
+            <div class="list-item" id="edit-list">
+                <input type="checkbox" class="checkbox">
+                <input type="text" class="desc" value="${item.description}">
+            </div>
+            <i class="fa-solid fa-ellipsis-vertical" id="drag"></i>
+            <i class="fa-regular fa-trash-can hidden" id="delete"></i>
+        </li>        
+        `;
+    todoItems.innerHTML += content;
+  });
+
+  // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ]  Kyndly implement a functionality to update the indexes of the items when you delete a task.

Notice in the image below that the index  should be updated when an item is deleted, in this case the index of the first item is 2 and it should be 1, also please take into account that all the other indexes should be updated as well. 

![image](https://user-images.githubusercontent.com/95243996/201174181-1a655671-f1b9-4d04-8601-89907a72660e.png)

![image](https://user-images.githubusercontent.com/95243996/201174561-79feb39d-23e8-463b-9fc7-d72ba3358d7e.png)